google-cloud-storage/src/main/java/com/google/cloud/storage/GrpcStorageOptions.java

```diff
@@ -660,6 +914,12 @@ public boolean isAttemptDirectPath() {
       return false;
     }
 
+    /** @since 2.41.0 This new api is in preview and is subject to breaking changes. */
+    @BetaApi
+    public boolean isEnableMetrics() {
+      return true;
```

Comment:
This is meant to be a default value, so that the `private boolean enableMetrics = GrpcStorageDefaults.INSTANCE.isEnableMetrics();` returns true by default. I was following what `isAttemptDirectPath()` does. Is that not correct?

Reply:
Oh, this is in `GrpcStorageDefaults` my mistake. I thought it was on `GrpcStorageOptions` itself. In that case, returning true here is fine.